gtsam/sfm/tests/testShonanAveraging.cpp

```diff
@@ -121,7 +121,7 @@ TEST(ShonanAveraging3, tryOptimizingAt4) {
 }
 
 /* ************************************************************************* */
-TEST(ShonanAveraging3, MakeATangentVector) {
+TEST(ShonanAveraging3, MakeATangentVectorValues) {
```

Comment:
Looking back on this, I don't think this was a great test, as the Hat thing is not needed. I created a new "before" test, please use this to make the new test, making sure the expected vectorts are exactly the same:
```c++
  Vector9 v;
  v << 1, 2, 3, 4, 5, 6, 7, 8, 9;
  Vector expected0(10), expected1(10), expected2(10);
  expected0 << 0, 3, -2, 1, 0, 0, 0, 0, 0, 0;
  expected1 << 0, 6, -5, 4, 0, 0, 0, 0, 0, 0;
  expected2 << 0, 9, -8, 7, 0, 0, 0, 0, 0, 0;
  const Vector xi_0 = ShonanAveraging3::MakeATangentVector(5, v, 0);
  const Vector xi_1 = ShonanAveraging3::MakeATangentVector(5, v, 1);
  const Vector xi_2 = ShonanAveraging3::MakeATangentVector(5, v, 2);
  EXPECT(assert_equal(expected0, xi_0));
  EXPECT(assert_equal(expected1, xi_1));
  EXPECT(assert_equal(expected2, xi_2));
```